packages/compass-e2e-tests/tests/global-preferences.test.ts

```diff
@@ -168,6 +169,26 @@ describe('Global preferences', function () {
     }
   });
 
+  it('allows setting readOnly: true and reflects that on the crud view', async function () {
+    await fs.writeFile(path.join(tmpdir, 'config'), 'readOnly: true\n');
+    // No settings modal opened for Isolated edition
+    const compass = await beforeTests();
```

Comment:
… but this isn’t isolated, so there _should_ be a settings modal, right? I’m not sure, maybe we just need to drop the comment(s) here after the recent welcome modal refactor.